### DISCUSSION TITLE
Cannot access account.

### DISCUSSION THREAD

**Discussion by mohdnoorlizam78** [community/community#38390] [Other Feature Feedback, Questions, & Ideas] [answered]
I can't access my other account because forgot password. Actually, my email also not valid anymore. How can I reset my password if my email is not valid? Anyone can help me?

**Comment by radar-blip** [comment, marked as answer]
You can try asking for help here: [Support page](https://support.github.com/contact/account)

**Reply by mohdnoorlizam78** [reply]
Thank you sir.

**Reply by radar-blip** [reply]
You're welcome!

**Reply by Barlaajay** [reply]
Ok Tq Sur

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mohdnoorlizam78 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
